module/VuFind/src/VuFind/Security/CspHeaderGenerator.php

```diff
@@ -100,10 +101,6 @@ public function getHeader()
             ) {
                 $sources[] = "'nonce-$this->nonce'";
             }
-            // Add report-uri header for backwards compatibility
-            if ($name == 'report-to') {
-                $cspHeader->setDirective('report-uri', $sources);
-            }
```

Comment:
Maybe worth adding a TODO checkbox with a changelog note to account for this change once we merge this.

Reply:
Good idea.  I added the warning.  I don't want to add the implied backwards compatibility since it will not actually work in the large majority of browsers.